modules/annotation-processor/src/main/java/org/ballerinalang/natives/annotation/processor/ConstructProviderClassBuilder.java

```diff
@@ -65,8 +65,8 @@
  */
 public class ConstructProviderClassBuilder {
     
-    private static final String SERVICES = "services" + File.separator;
-    private static final String META_INF = "META-INF" + File.separator;
+    private static final String SERVICES = "services/";
+    private static final String META_INF = "META-INF/";
```

Comment:
using forward-slash instead of file separator, as this would be converted to system specific file-separator by the javax.annotation.processing.Filer,  by which this constant is used.